### PR TITLE
Updated readme with usability tips

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
Applying some cool hacks to show how users can run "azure vm docker" within a container with one simple `alias` command.

cc: @johngossman @kmouss @devigned

### CODE REVIEW

File: 0.9.2/README.md [hunk 1]
```diff
@@ -2,4 +2,40 @@
...
+$ docker run -it -v /.azure:/root/.azure microsoft/azure-cli
+```
+
+### More practical usage methods
+
+Instead of running `docker run` command over and over again to execute Azure CLI
+tasks, you can alias the command to `azure` (in Unix shells) as follows:
+
+```bash
+$ alias azure="docker run -it --rm -v /.azure:/root/.azure microsoft/azure-cli azure"
```
Comment (paulmey) [inline 0.9.2/README.md]: Does this assume that you're running as root? Not sure if that's the wisest advice... :-)

File: 0.9.2/README.md [hunk 2]
```diff
@@ -2,4 +2,40 @@
...
+```
+
+then run commands just like:
+
+```bash
+$ azure vm list
```
Comment (paulmey) [inline 0.9.2/README.md]: maybe we should mention that this is limited to commands that don't use files from the local filesystem (e.g. `azure group deployment create -f ...`)

Comment (ahmetalpbalkan) [inline 0.9.2/README.md]: hmm, figuring that out left as an exercise to user :P ? maybe this change is not needed after all.

File: 0.9.2/README.md (hunk 1 above)
Comment (ahmetalpbalkan) [inline 0.9.2/README.md]: @paulmey erm, yes. I guess the docker extension also installs and starts docker as root (coz many people do it really)... suggestions for any other paths?

**Comment by devigned** [comment]
Closing based on age. Please reopen if we want to pursue again.

**Comment by ahmetalpbalkan** [comment]
Sounds good.

**Pull request closed without merging**